include/opcode/riscv-opc.h

```diff
@@ -929,6 +929,12 @@
 #define MASK_ABS      0xFFF0707F
 #define MASK_SLET     0xFE00707F
 #define MASK_ADDN     0xC000707F
+/* Immediate Branching */
+#define MATCH_BNEIMM    0x00003063
+#define MATCH_BEQIMM    0x00002063
+
+#define MASK_BNEIMM     0x0000707f
+#define MASK_BEQIMM     0x0000707f
```

Comment:
As mentioned in the meeting these MATCHES and MASKS have been changed. When I rebased someone had used the same names as a number of ours. All CORE-V MATCHES and MASKS have been renamed to MATCH_CV_xxx or MASK_CV_xxx. Please see here https://github.com/jessicamills/corev-binutils-gdb/blob/jessica-development-rebase/include/opcode/riscv-opc.h.